src/app/common/components/BeaconchaLink/BeaconchaLink.tsx

```diff
@@ -16,9 +16,8 @@ const BeaconchaLink = ({ width, height, address, network }: BeaconchaLinkProps)
   const classes = useStyles();
   const stores = useStores();
   const applicationStore: ApplicationStore = stores.Application;
-  const imgSrc = `/images/beaconcha${applicationStore.isDarkMode ? '-white' : ''}.svg`;
+  const imgSrc = `/images/beaconcha${applicationStore.isDarkMode ? '-white' : '-new'}.svg`;
```

Comment:
maybe better idea is to name those svg files as `..-light.svg` and `...-dark.svg` ?

Reply:
done